<ui version="4.0" >
 <class>StatusOptionsWidgetClass</class>
 <widget class="QWidget" name="StatusOptionsWidgetClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>StatusOptionsWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QTableWidget" name="tbwRules" >
     <property name="selectionMode" >
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbtAdd" >
       <property name="text" >
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtDelete" >
       <property name="text" >
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>tbwRules</tabstop>
  <tabstop>pbtAdd</tabstop>
  <tabstop>pbtDelete</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
